shopinvader_search_engine/models/shopinvader_backend.py

```diff
@@ -35,29 +41,68 @@ def clear_index(self):
         return True
 
     def add_missing_index(self):
+        _logger.warning("DEPRECATED: add_missing_index, use action_add_missing_indexes")
+        self.action_add_missing_indexes()
+
+    def action_add_missing_indexes(self):
         self.ensure_one()
-        ir_models = self._get_default_models()
-        index_obj = self.env["se.index"]
-        ir_export_obj = self.env["ir.exports"]
         if not self.se_backend_id:
             return
-        for model in ir_models:
+        self._add_missing_indexes()
```

Comment:
```suggestion
       return self._add_missing_indexes()
```

Reply:
this is an action called from a btn you should not return records